### ISSUE TITLE
Feature Request: Make mistakes more obvious

### ISSUE THREAD

**Issue by DHNishi** [supermedium/moonrider#43]
It's hard to notice when a beat is whiffed or if a cut is in the wrong direction.

Other rider-type games make this mistake very noticeable.

**Comment by ngokevin** [comment]
Thanks. We had this, but removed it until we get hits more reliable to mask the game's detection issues, else it'd be pretty frustrating! Will perhaps add again once we get things more solid.

**Comment by JustinB4** [comment]
The detection seems a lot better now.  It would be cool if there was a debug option to make the correct notes vibrate stronger (or longer) or possibly make missed notes more obvious (or reason for miss).  I did notice that correct notes are a bit more obvious now with the ripple.  That's nice.

**Comment by ngokevin** [comment]
yeah a ripple indicates a "good/perfect" hit in regards to your accuracy (sufficient speed + direction regard to arrow).

i can maybe turn the score text red, or flash the road red. wondering if the latter would be too distracting
